app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,57 @@
+class InventoriesController < ApplicationController
+  before_action :inventory, only: %i[show edit update destroy]
+
+  def index
+    @user = User.find(params[:user_id])
+    @inventories = Inventory.includes(:user).where(user: params[:user_id])
+  end
```

Comment:
Hi @Theophileaseh 
According to the wireframe, you just need to show a list of all inventories. There is no need of showing user params. Because inventories do not involve users at all. 

![image](https://user-images.githubusercontent.com/62090797/187352351-93bf2608-719d-4f6d-abfa-93bf8525b9fe.png)